tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+# from pathlib import Path
```

Comment:
Seems that this file is left over

Reply:
I just intentionally left this file. These are another configuration styles for setting Thundra Agent. If someone would like to set Thundra by one of these method, uncommenting one of these configuration style and setting appropriate values is only things they have to do.